Project.toml

```diff
@@ -5,6 +5,7 @@ version = "0.1.4"
 
 [deps]
 Distances = "b4f34e82-e78d-54a5-968a-f98e89d6e8f7"
+GeoStatsBase = "323cb8eb-fbf6-51c0-afd0-f8fba70507b2"
```

Comment:
You shouldn't need this package as an explicit dependence. Perhaps you need to locally dev a global environment with all dev versions like:

```julia
] activate @geo 
] dev GeoStatsBase
] dev GeoClustering
```

That way you have the dev version of all packages in a global environment `@geo` that you can activate whenever you are working on the GeoStats.jl stack.
```

Reply:
Please remove the GeoStatsBase.jl dependency.